### ISSUE TITLE
Liquidsoap plugin

### ISSUE THREAD

**Issue by meyskens** [Innovate-Technologies/DJ#2]
In early builds DJ used Liquidsoap to mix and send out streams. The codebase remained but did not get tested. This will be the audio engine for the open source version.

**Comment by meyskens** [comment]
Due limited time this will currently not be built by us. Feel free to send a PR.

**Comment by jersobh** [comment]
Hi there... I'll take a try :) :+1:

**Comment by meyskens** [comment]
@jersobh nice :) there is already some code that uses telnet to control LS. I am not sure if I have added the .liq script in there but I have it somewhere on our old repo.

**Comment by jersobh** [comment]
Sure :) I've emailed you yesterday. Sorry to bother you that much :x 
I'm trying to join webrtc + liquidsoap. And I'm having a bad time doing that. 
I just saw your github profile and I thought "omg". You have some amazing job there. 
Cheers and thank you so much!

**Comment by meyskens** [comment]
<img width="53" alt="schermafbeelding 2017-03-31 om 20 04 41" src="https://cloud.githubusercontent.com/assets/1625272/24563024/4af829a6-164d-11e7-95c2-5c8feff57855.png">
If you expect me to see emails you now know why I do miss some ;) 

Do you mean webrtc as in the voip protocol? As an input source I guess?

**Comment by jersobh** [comment]
Exactly! But liquidsoap cannot cast streams directly. Right now I'm trying this approach: https://github.com/muaz-khan/RecordRTC/tree/master/RecordRTC-over-Socketio
To make it a usable audio source.

**Comment by meyskens** [comment]
hmm, can't you make a harbor-like plugin? I was looking into LS a lot before I decided to give our own audio engine a try. As much as we love open source we don't plan to open that part up as it is far more efficient and giving it away could kill our company. 

What we plan to use there is what we're working on: ORP or Open Radio Protocol. It is basically a open source rewrite of the SHOUTcast v2 protocol on top of websockets and more modern languages, meaning metadata include, multiplexing,... I have simple demo code somewhere but it is far from ready.

**Comment by jersobh** [comment]
I know that, I also love open sourcing, but some things are made to be kept in secret :)
That's awesome! SHOUTcast is so much outdated. I'll let you know of any progress here.
Thank you so much
